src/ui/public/partials/paginate_controls.html

```diff
@@ -1,3 +1,7 @@
+<div class="top-link" ng-if="linkToTop">
+  <a ng-click="paginate.goToTop()">^ Top</a>
+</div>
```

Comment:
We can simplify this markup by just using the anchor element. Could you also change this markup to incorporate a new class we've added, as well as a special selector to use in tests? And let's take out the caret and tweak the copy a bit, too:

```html
<a
  class="kuiLink"
  ng-if="linkToTop"
  ng-click="paginate.goToTop()"
  data-test-subj="paginateControlsLinkToTop"
>
  Scroll to top
</a>
```